prepare.cmd

```diff
@@ -0,0 +1,81 @@
+@echo off
+
+net session >nul 2>&1
+if %ERRORLEVEL% GTR 0 (
+    echo Administration rights required for using mklink.exe command!
+    echo You are NOT Administrator. Exiting...
+    timeout 10
+    exit /B 1
+)
+
+%~d0
+cd %~dp0
+
+if not exist Source goto wrong_dir
+
+echo Scan for git.exe in path...
+where /q git.exe
+if %ERRORLEVEL% GTR 0 (
+    echo git.exe command not found in path!
+    echo Please install from https://gitforwindows.org/
+    timeout 10
+    exit /B 1
+)
+
+echo Check JsonDataObjects...
+if not exist ..\JsonDataObjects git clone https://github.com/ahausladen/JsonDataObjects.git ..\JsonDataObjects
+
+echo Check OmniThreadLibrary...
+if not exist ..\OmniThreadLibrary git clone https://github.com/gabr42/OmniThreadLibrary.git ..\OmniThreadLibrary
+
+echo Check Spring4D...
+if not exist ..\spring4d git clone https://bitbucket.org/sglienke/spring4d.git ..\spring4d
```

Comment:
change this to resolve the const issue

if not exist ..\spring4d git clone --branch develop https://bitbucket.org/sglienke/spring4d.git ..\spring4d